pkg/kubectl/describe.go

```diff
@@ -2068,3 +2067,28 @@ func (fn typeFunc) Describe(exact interface{}, extra ...interface{}) (string, er
 	}
 	return s, err
 }
+
+// printLabelsMultiline prints multiple labels with a proper alignment.
+func printLabelsMultiline(out io.Writer, title string, labels map[string]string) {
```

Comment:
Probably both have to be controlled independently - generally we control our own indentation inside any nested template (containers are an example), and only rely on the tab writer at the top level.  Since labels can be part of the container section you may need two printers - one in the "formatLabel" form and one in the "works inside printContainer" form